src/debug_functions.h

```diff
@@ -1,6 +1,19 @@
 #pragma once
 
+#include "aoapplication.h"
+
 #include <QString>
 
 void call_error(QString message);
 void call_notice(QString message);
+
+class debug_functions : public QObject
+{
+  Q_OBJECT
+
+public:
+  explicit debug_functions(AOApplication *parent);
+  AOApplication *ao_app;
+
+  void call_notice_reconnect(QString message);
+};
```

Comment:
What's the reasoning behind making this an object over a simple function call? I don't see a reason as to why it would be.

Reply:
Simply add a `bool prompt_choice(QString message)` which either allow you to accept or cancel. That way you keep it within a singular function and can use it elsewhere within the code. Additionally, the return value will make it easy on whatever reconnecting is wanted by the user or not.